arbitrum-docs/node-running/how-tos/running-an-orbit-node.mdx

```diff
@@ -59,7 +59,7 @@ This how-to provides step-by-step instructions for running an Orbit node on your
 
   ```
 
-  - Note for Anytrust chains, it is essential to add `--node.data-availability.enable` to the command as well.
+  - Note for Anytrust chains, it is essential to add `--node.data-availability.enable` to the command and add `--node.data-availability.rest-aggregator.urls=<Your DAS rest endpoints>`/`--node.data-availability.rest-aggregator.urls=<A url can return a list of your das rest endpoints>` as well.
```

Comment:
```suggestion
  - Note for Anytrust chains, it is essential to add `--node.data-availability.enable`, and `--node.data-availability.rest-aggregator.urls=<A list of DAS REST endpoints>` or `--node.data-availability.rest-aggregator.online-url-list=<A url that returns a list of the DAS REST endpoints>` to the command or configuration.
```

Proposing some reformatting, and changing the second parameter to show both available options (modify if you see anything wrong!) :pray: